src/cmd/promremotebench/query.go

```diff
@@ -427,7 +476,11 @@ func (q *queryExecutor) validateQuery(dps Datapoints, data []byte) bool {
 	}
 
 	if matches == 0 {
-		q.Logger.Error("no values matched at all.")
+		q.Logger.Error("no values matched at all.",
```

Comment:
nit: Could be worth doing something like `logger := q.Logger.With(zap.String("hostname", hostname))` at the start of this function, then use that instead of `q.Logger`? Then should get the hostname added to other log statements for free